lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -68,17 +69,21 @@ getClosestInsertSliceOfResult(OpResult result) {
       sliceOp =
           dyn_cast<OffsetSizeAndStrideOpInterface>(useOfResult.getOwner());
     } else if (auto yieldOp = dyn_cast<scf::YieldOp>(useOfResult.getOwner())) {
-      if (auto loop = dyn_cast<LoopLikeOpInterface>(yieldOp->getParentOp()))
+      if (isa<LoopLikeOpInterface, scf::IfOp>(yieldOp->getParentOp())) {
```

Comment:
Will `RegionBranchInterface` better than `scf::IfOp` here?

Reply:
Thanks for this advice! Changed.